app/controllers/admin/api_docs/account_api_docs_controller.rb

```diff
@@ -27,10 +31,18 @@ def redirect_to_service_scope
   end
 
   def service
-    api_docs_service.service
+    api_docs_service.owner
```

Comment:
I guess this is true only if the `ApiDocs::Service` is owned by a `Service`, right? Maybe keep
```suggestion
    api_docs_service.service
```
...and solve in the model? e.g.
```ruby
class ApiDocs::Service < ApplicationRecord
  def owner_by_service?
    owner_type == 'Service'
  end

  def service
    owner if owner_by_service?
  end
end
```